src/user/UserRouter.ts

```diff
@@ -1,50 +1,32 @@
-import { Request, Response, NextFunction, Router } from 'express';
+import { Request, Response, Router } from 'express';
 import UserService from './UserService';
+import { check, validationResult } from 'express-validator';
 
 const router = Router();
 
-const validateUsername = (req: Request, res: Response, next: NextFunction) => {
-  //
+router.post(
+  '/api/1.0/users',
+  check('username').notEmpty().withMessage('Username cannot be null'),
+  check('email').notEmpty().withMessage('E-mail cannot be null'),
+  async (req: Request, res: Response) => {
+    const errors = validationResult(req);
 
-  const requestBody = req.body as { password: string; username: string; email: string };
+    if (!errors.isEmpty()) {
+      const validationErrors: { [key: string]: string } = {};
 
-  const { username } = requestBody;
+      errors.array().forEach((error) => {
+        validationErrors[error.param] = error.msg;
```

Comment:
Creating a custom validation error object response for user to see it will be 

```js
{
  validationErrors: {
    username: 'Username cannot be null',
    email: 'E-mail cannot be null'
  }
}
```